corona-warn-app/cwa-app-android
# Validation start screen (EXPOSUREAPP-8030)

Validation start screen + multiple urls in one text extension (big cheers to @d4rken 🍺 )

<img width="223" alt="Screenshot 2021-07-02 at 15 54 40" src="https://user-images.githubusercontent.com/64849422/124277660-f8ea4700-db4d-11eb-8bb8-c3ec4dcbc5cb.png">

How to test:
1) Scan test certificate
2) Go to certificate details
3) Press "Gültigkeit prüfen" at the bottom of the screen (would be replaced with correct button in upcoming PR)
4) Check new screen

TODO: Wire things up with some server check magic
